### PR TITLE
Add endpoint

### PR DESCRIPTION
**Pull request by JoseLion**
- Organized the app so the file ./bin/www.js sets up an begins the listening of the server (use this file to run the app)
- Added sequelize and a connection to the database (it can be accessed with: `import sequelizeConn from 'Core/sequelizeConn';`)
- Added a 'User' model (it can be accessed with: `import User from 'Models/User';`)
- Added sequelize migrations and seeders
- Added an endpoint which return all users in database (/api/users)
- Added a function which add a test user programmatically (uncomment line 11 of ./src/app.js)
- Added BabelJS as transpiler so some other ES6 features are available (import and export specially)
- Added babel-module-resolver to create alias to folders for imports (improves coding organization and prevents from typing large absolute paths several times)
- The app can now be run without the database connection environment variable

### CODE REVIEW

File: backend/src/app.js [hunk 1]
```diff
@@ -1,125 +1,80 @@
...
-    if (valid_token === token) {
-      return username;
-    } else {
-      return null;
-    }
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.static("../frontend/build"));
+
+app.get('/api/users', async (request, response) => {
```
Comment (po5i) [inline backend/src/app.js]: Can you add how to test this endpoint to the README?

File: backend/bin/www.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require('babel-register');
+
+const http = require('http');
+const app = require("../src/app").default;
+
+if (!process.env.DB_CONN_NANOCHAT) {
```
Comment (po5i) [inline backend/bin/www.js]: As discussed, please document all the necessary steps to start the server locally

**Review by po5i — CHANGES_REQUESTED** [review]

**Review by po5i — APPROVED** [review]
LGTM 🎉 🚢

**Review by sestrella — APPROVED** [review]
@JoseLion LGTM nice job!

**Pull request merged**